### ISSUE TITLE
Bulkrax: default mappings

### ISSUE THREAD

**Issue by crisr15** [notch8/utk-hyku#144]
# Summary

Kirk had to implement bulkrax mappings for the Collection metadata. If the header is named the same as the property, you should not have to set the mappings in bulkrax config. 

This ticket is to figure out why we have to, but we're unblocked so this can be lower priority.

# Acceptance Criteria

- [ ]

# Screenshots or Video

# Testing Instructions

# Notes

Confused how to fill this out? Check out the [playbook](https://playbook-staging.notch8.com/en/notch8/process/issues)

**Comment by DiemBTran** [comment]
Related MRs:

- [I215 collection with bulkrax](https://gitlab.com/notch8/utk-hyku/-/merge_requests/146)
